social_network_api/src/main/java/com/danit/socialnetwork/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.danit.socialnetwork.exception;
+
+import com.danit.socialnetwork.exception.user.HeaderPhotoNotFoundException;
+import com.danit.socialnetwork.exception.user.PhotoNotFoundException;
+import com.danit.socialnetwork.exception.user.UserNotFoundException;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@Log4j2
+@ControllerAdvice
+public class GlobalExceptionHandler {
+  @ExceptionHandler
+  public ResponseEntity<ErrorResponse> catchPhotoNotFoundException(PhotoNotFoundException exception) {
+    ErrorResponse error = new ErrorResponse();
+    error.setStatus(HttpStatus.NOT_FOUND.value());
+    error.setMessage(exception.getMessage());
+    error.setTimeStamp(System.currentTimeMillis());
+    return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
+  }
+
+  @ExceptionHandler
+  public ResponseEntity<ErrorResponse> catchUserNotFoundException(UserNotFoundException exception) {
+    ErrorResponse error = new ErrorResponse();
+    error.setStatus(HttpStatus.NOT_FOUND.value());
+    error.setMessage(exception.getMessage());
+    error.setTimeStamp(System.currentTimeMillis());
+    return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
+  }
+
+  @ExceptionHandler
+  public ResponseEntity<ErrorResponse> catchHeaderPhotoNotFoundException(HeaderPhotoNotFoundException exception) {
+    ErrorResponse error = new ErrorResponse();
+    error.setStatus(HttpStatus.NOT_FOUND.value());
+    error.setMessage(exception.getMessage());
+    error.setTimeStamp(System.currentTimeMillis());
+    return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
```

Comment:
You can create one method for all these exceptions

Reply:
@miegami I thought about this I can do to add just one AppUserError. If I just return informaton it is ok, but for the future if somebody would like to process some of them separately it better to have them separately.

Reply:
Done, if somebody would need this he would add ExceptionHandler for definite Exception

Reply:
I mean that you can write something like @ExceptionHandler({FirstException.class, SecondException.class})